bvinson23/finish-finder
# Bv editboard

# Description
User can edit a board when pressing the pencil button on a board.
## Type of change
# Testing Instructions
Click the pencil on a board, then select a new finish or all new finishes for the board, the new selections should show up on the right side, then click save, the board will be updated.
# Checklist:
